src/Ztm.Zcoin.NBitcoin/ZcoinTxIn.cs

```diff
@@ -17,11 +17,16 @@ public ZcoinTxIn(ZcoinConsensusFactory consensusFactory)
             this.consensusFactory = consensusFactory;
         }
 
+        bool IsType(ZcoinOpCode opCode)
```

Comment:
The behavior of this method will work only on our custom script due to in Bitcoin the OP that define the script behavior will be the last one.

Reply:
then I will change name to `IsStartsWith`